database/migration/aruha-334-rename-subscription-fields.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE zn_data.subscription
+SET s_subscription_object = jsonb_set(s_subscription_object, '{consumer_group}', '"default"')
```

Comment:
Are you sure we can use it for 9.4? I could not managed to find it https://www.postgresql.org/docs/9.4/static/functions-json.html